apps/docs/src/app/platform/component-docs/platform-button/platform-button-docs.component.html

```diff
@@ -35,23 +35,35 @@
     Button State
 </fd-docs-section-title>
 <description>
-    <code>[disabled]="true"</code> sets the button disabled
+    'disabled' and 'selected' states can be achieved through 'disabled' / 'ariaDisabled' and 'ariaSelected'
+    attributes.
 </description>
 <component-example>
     <fdp-button-state-example></fdp-button-state-example>
 </component-example>
 <code-example [exampleFiles]="buttonHtmlState"></code-example>
 
-<!-- <separator></separator> -->
+<separator></separator>
 
-<!-- <fd-docs-section-title [id]="'Button-truncate'" [componentName]="'button'">
+<fd-docs-section-title [id]="'Button-truncate'" [componentName]="'button'">
     Button with long text truncated
 </fd-docs-section-title>
 <description>
-    When user specifices with width of the button then text will start truncating to accomdate it self in the specified
+    When user specifices width of the button then text will start truncating to accomadate it self in the
+    specified
     width as shown below
 </description>
 <component-example>
     <fdp-button-truncate-example></fdp-button-truncate-example>
 </component-example>
-<code-example [exampleFiles]="buttonHtmlTruncate"></code-example> -->
+<code-example [exampleFiles]="buttonHtmlTruncate"></code-example>
+
+<separator></separator>
+
+<description>The inputs for <code>button</code> will be translated into classes attributed to fundamental-styles
```

Comment:
not needed of this paragraph as it is covered before

Reply:
Removed

Reply:
removed